docs/wiki/blocks/block-body.md

```diff
@@ -0,0 +1,70 @@
+# Block Body
```

Comment:
This should be called Transaction Body, as that is the type of both a Block body and Transaction body.
The document should be renamed to something like docs/wiki/transactions/transaction-body.md

Reply:
Updated.